Makefile

```diff
@@ -147,7 +147,7 @@ test: $(GOBUILDER) | $(DIST)
 
 clean:
 	rm -rf $(DIST) pkg/pillar/Dockerfile pkg/qrexec-lib/Dockerfile pkg/qrexec-dom0/Dockerfile \
-		images/installer.yml $(ROOTFS_YML_xen) $(ROOTFS_YML_acrn)
+		pkg/mkpersist/Dockerfile images/installer.yml $(ROOTFS_YML_xen) $(ROOTFS_YML_acrn)
```

Comment:
As we've briefly discussed today, lets drop this dependency on gpttools and simplify this for now to be self-contained.

Reply:
will do